http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceV2InterpSpec.scala

```diff
@@ -44,15 +44,19 @@ class RuntimeServiceV2InterpSpec extends AnyFlatSpec with LeonardoTestSuite with
   val wsmDao = new MockWsmDAO
 
   // used when we care about queue state
-  def makeInterp(queue: Queue[IO, LeoPubsubMessage], wsmDao: WsmDao[IO] = wsmDao) =
-    new RuntimeV2ServiceInterp[IO](serviceConfig, whitelistAuthProvider, wsmDao, mockSamDAO, queue)
+  def makeInterp(queue: Queue[IO, LeoPubsubMessage],
+                 wsmDao: WsmDao[IO] = wsmDao,
+                 dateAccessedQueue: Queue[IO, UpdateDateAccessMessage] = QueueFactory.makeDateAccessedQueue()
+  ) =
+    new RuntimeV2ServiceInterp[IO](serviceConfig, whitelistAuthProvider, wsmDao, mockSamDAO, queue, dateAccessedQueue)
```

Comment:
NIT, I know this existed before your change, but would it be difficult to change `whitelistAuthProvider` to `allowlistAuthProvider`? We should try to clean up some of leo's language whenever we see it